server/src/main/java/com/objectcomputing/checkins/services/memberprofile/MemberProfileServicesImpl.java

```diff
@@ -1,7 +1,7 @@
 package com.objectcomputing.checkins.services.memberprofile;
 
 
-import com.objectcomputing.checkins.security.InsufficientPrivelegesException;
+import com.objectcomputing.checkins.services.exceptions.PermissionException;
```

Comment:
This has been moved to com.objectcomputing.checkins.exceptions.PermissionException. Try pulling the latest.